src/alttxt/models.py

```diff
@@ -42,6 +43,10 @@ class PlotModel(BaseModel):
     histograms: list # of float
     wordclouds: list # of float
 
+class metaDataModel(BaseModel):
```

Comment:
It's convention that classes have their first letter capitalized: 
```suggestion
class MetaDataModel(BaseModel):
```